standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -1317,6 +1318,7 @@ struct GetTableRequest {
   3: optional ClientCapabilities capabilities,
   4: optional string catName,
   6: optional string validWriteIdList
```

Comment:
Huh. thrift didn't complaint about the comma :(. Added the comma.

I am not sure about 5 though. I just added the next field with the next number 7. thrift seems to be Ok with that. It ignores the number 5 in geneated GetTableRequest.java as well. Do you think we should fix this?

Reply:
Missing 5 won't have any impact. But still good to fix it as we seen it. Moreover, I think I added that code earlier and missed it... :)

Reply:
Thrift changes are tricky to make and might lead to other problems, we do not know. Since it's not exactly related to stats change and it's not causing problems to the feature, I would like to leave it aside.